src/Strategy/Maven/Common.hs

```diff
@@ -22,6 +25,43 @@ data MavenDependency = MavenDependency
 mavenDependencyToDependency :: MavenDependency -> Dependency
 mavenDependencyToDependency MavenDependency{..} = dependency
 
+filterMavenSubmodules :: Set Text -> Set Text -> Graphing MavenDependency -> (Graphing MavenDependency)
```

Comment:
I didn't add a comment because I thought that the function name captured the its functionality pretty well. Was there confusion about the input parameters that are being passed? If so, I'll add a comment.